scripts/run-tests.sh

```diff
@@ -1,7 +1,7 @@
-# ./run-tests --help
+#!/usr/bin/env bash
+set -eo pipefail
```

Comment:
Did you encounter a problem without `pipefail`?

Reply:
set -e doesnt work in pipes so this adds some extra robustness

the error was shebang not being in the first line

Reply:
I see